proto/cometbft/crypto/v1/keys.proto

```
@@ -0,0 +1,19 @@
+syntax = "proto3";
+package cometbft.crypto.v1;
+
+option go_package = "github.com/cometbft/cometbft/api/cometbft/crypto/v1";
+
+import "gogoproto/gogo.proto";
```

Buf (1.54.0)

import "gogoproto/gogo.proto": file does not exist

(COMPILE)